res/TensorFlowLiteRecipes/Quant_Split_Add_000/test.rule

```diff
@@ -0,0 +1,11 @@
+# To check mixed-precision quantization for multiple output node.
+# Split: int16, Add: u8
+# Quantize Ops are inserted before Split and after all Split output nodes.
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "INPUT_INT16"             $(tensor_dtype ifm) '=' UINT8
```

Comment:
```suggestion
RULE    "INPUT_UINT8"             $(tensor_dtype ifm) '=' UINT8
```

Reply:
Thanks, I will fix it in pr in `res` part (https://github.com/Samsung/ONE/pull/8673#discussion_r831825495)